README.md

```diff
@@ -1,32 +1,12 @@
 
-PK-Sim® uses building blocks that are grouped into Individuals, Populations, Compounds, Formulations, 
-Administration Protocols, Events, and Observed Data. Building blocks from these groups are combined to produce a model. 
-The advantage of building blocks is that they can be reused. 
-For example, after having established a model for a drug after single dose intravenous administration to an animal species, 
-just substitute the individual by a suitably parameterized virtual human population and obtain a first in man simulation model. 
-Further substitute the formulation, to obtain a controlled-release per oral simulation model, substitute the protocol 
-to obtain a multiple dose simulation model, or substitute the compound to obtain a simulation model for another drug.
+PK-Sim® uses building blocks that are grouped into [**Individuals**](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-creating-individuals), [**Populations**](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-creating-populations), [**Expression Profiles**](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-profile), [**Compounds**](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow), [**Formulations**](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-formulations), [**Administration Protocols**](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-administration-protocols), [**Events**](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-events), **Observers** and [**Observed Data**](https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/import-edit-observed-data). Building blocks from these groups are combined to produce a [model](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations). The advantage of building blocks is that they can be reused. For example, after having established a model for a drug after single dose intravenous administration to an animal species, just substitute the individual by a suitably parameterized virtual human population and obtain a first in man simulation model. Further substitute the formulation, to obtain a controlled-release per oral simulation model, substitute the protocol to obtain a multiple dose simulation model, or substitute the compound to obtain a simulation model for another drug. 
```

Comment:
Also replaced the outdated description with the actual one (copied from the README in the Suite repo)